README.md

```diff
@@ -774,6 +774,29 @@ Translations of the guide are available in the following languages:
   end
   ```
 
+* <a name="avoid-repeating-aliased-module-names"></a>
+  If you plan to alias your modules, avoid using the same name consecutively.
+  This avoids any problems with [Conflicting Aliases]
+  for the module.
+  <sup>[[link](#avoid-repeating-aliased-module-names)]</sup>
+
+  ```elixir
+  # not preferred
+  defmodule Todo.Todo do
+    alias Todo.Todo
+  end
+
+  # somewhat preferred
```

Comment:
Can you please help me with a better example for this? I was just following the original thread and the issue so not sure what a good example would be. To be honest, I would never name something consecutively like the example so really couldn't think of a use case where that is even possible.